src/Resources/public/js/events.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+pimcore.events.cmf = {
```

Comment:
please keep it consistent like other bundle event names so using something like `pimcore.events.onCmfMenuReady`. please see https://github.com/pimcore/perspective-editor/blob/42675e3fb0b58e3f3ea33edd082434189de7a0f2/src/Resources/public/js/pimcore/perspective/events.js#L18